helper/git-extra-utility

```diff
@@ -17,3 +17,12 @@ git_extra_default_branch() {
         echo "main"
     fi
 }
+
+#
+# get processor count, default 2
+#
+git_extra_get_nproc() {
+    if ! (nproc || getconf NPROCESSORS_ONLN || getconf _NPROCESSORS_ONLN || grep -c ^processor /proc/cpuinfo) 2>/dev/null; then
+        printf '%s\n' 2
```

Comment:
Can we allow the caller to push the default value?

Reply:
:+1: